template/src/ontology/Makefile.jinja2

```diff
@@ -935,9 +926,9 @@ $(ONT)-baseplus.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC) $(IMPORT_FILES)
 		{% if project.release_date -%} --annotation oboInOwl:date "$(OBODATE)" {% endif -%}--output $@.tmp.owl && mv $@.tmp.owl $@
 {% endif -%}
 
-{% if 'editors' in project.release_artefacts or project.primary_release == 'editors' -%}
-# editors: All the axioms as they are editted by the editors, excluding reasoning. This is currently the same as "base". Only to be used experimentally (may disappear in future ODK releases).
-$(ONT)-editors.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC)
+{% if 'baselite' in project.release_artefacts or project.primary_release == 'baselite' -%}
+# baselite: All the axioms as they are editted by the editors, excluding reasoning. This is currently the same as "base". Only to be used experimentally (may disappear in future ODK releases).
+$(ONT)-baselite.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC)
```

Comment:
Due to request by @dosumis I renamed the `editors` release to `baselite`. This release corresponds to the editor provided axioms "as they are", without any processing, reasoning, redundancy stripping etc.